building/userland_integration.rst

```diff
@@ -30,29 +30,41 @@ part of the `OP-TEE client tools <https://github.com/OP-TEE/optee_client>`_.
 
 Once OP-TEE has been compiled with the PKCS#11 TA, the client tools shared
 object has been built and the OP-TEE supplicant has been started, we can use
-``pkcs11-tool`` to initiate first communication with the emulated smart card.
-In the following, we assume that ``libckteec.so`` has been installed in
-``/usr/local/lib/libckteec.so``.
+``pkcs11-tool`` of the `OpenSC project <https://github.com/OpenSC/OpenSC>`_ to
+initiate first communication with the emulated smart card.  In the following,
+we assume that ``libckteec.so`` has been installed in
+``/usr/local/lib/libckteec.so``. For simplicity reasons, we define an alias to
```

Comment:
Actually in the QEMU environment (at least QEMUv8), it is `/usr/lib/libckteec.so.0`. Should this path be used instead?